ironfish/src/evm/evm.ts

```diff
@@ -13,6 +13,9 @@ import { EvmStateDB } from './database'
 export class IronfishEvm {
   stateManager: DefaultStateManager
   private vm: VM
+  // spending key
+  static spendingKey = '3333fff66a72eff15588d48d9b5404e3a20a4ab90e0dfcf4aa999f1855788da6'
+  static publicAddress = '2edbd0f3296a925ce373598294102c7179902bb5323b8bdcec54feabd10658d4'
```

Comment:
How will we use this key and address? Will the global account be signing with this key?

Reply:
I imagined that the mint descriptions will be generated with this key and the public address will own all the shielded assets. This way anyone can generate the mints. I suspect we will need to sign with this key too since the outputs will be generated from this account.